template/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MPL-2.0",
   "scripts": {
     "start": "j1-integration collect",
+    "visual": "j1-integration visualize",
```

Comment:
Thoughts on maybe making the script name `visualize` or perhaps even `graph`?

Reply:
I like graph, I'll change it to that.